proxy/proxy.go

```diff
@@ -371,7 +371,8 @@ func (e *proxyError) DialError() bool {
 }
 
 func (e *proxyError) NetError() net.Error {
```

Comment:
Lets remove `NetError()`, it is not used and `proxyError` is not exported

Reply:
done